From e26cead606fc3f860f073765615b8ca08ed8b346 Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Fri, 26 Aug 2016 15:36:13 -0400
Subject: [PATCH 35/58] ARM: dts: keystone: Disable 10gbe SerDes lane nodes by
 default

When 10gbe functionality needs to be enabled, the corresponding
SerDes lane nodes should be enabled explicitly together with
the SerDes node itself.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
Signed-off-by: Michael Scherban <m-scherban@ti.com>
Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 arch/arm/boot/dts/keystone-k2e-netcp.dtsi  | 4 ++--
 arch/arm/boot/dts/keystone-k2hk-netcp.dtsi | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/keystone-k2e-netcp.dtsi b/arch/arm/boot/dts/keystone-k2e-netcp.dtsi
index 473101b..a3a8d17 100644
--- a/arch/arm/boot/dts/keystone-k2e-netcp.dtsi
+++ b/arch/arm/boot/dts/keystone-k2e-netcp.dtsi
@@ -637,7 +637,7 @@ xgbe_serdes: phy@231e000 {
 	clock-names	= "fck";
 
 	xserdes_lane0: lane@0 {
-		status		= "ok";
+		status		= "disabled";
 		/*loopback;*/
 		#phy-cells	= <0>;
 		reg		= <0>;
@@ -647,7 +647,7 @@ xgbe_serdes: phy@231e000 {
 		tx-coeff	= <2 0 0 12 4>; /* c1 c2 cm att vreg */
 	};
 	xserdes_lane1: lane@1 {
-		status		= "ok";
+		status		= "disabled";
 		/*loopback;*/
 		#phy-cells	= <0>;
 		reg		= <1>;
diff --git a/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi b/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi
index 3fedae7..6abebb0 100644
--- a/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi
+++ b/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi
@@ -576,7 +576,7 @@ xgbe_serdes: phy@231e000 {
 	#size-cells	= <0>;
 
 	xserdes_lane0: lane@0 {
-		status		= "ok";
+		status		= "disabled";
 		/*loopback;*/
 		#phy-cells	= <0>;
 		reg		= <0>;
@@ -586,7 +586,7 @@ xgbe_serdes: phy@231e000 {
 		tx-coeff	= <2 0 0 12 4>; /* c1 c2 cm att vreg */
 	};
 	xserdes_lane1: lane@1 {
-		status		= "ok";
+		status		= "disabled";
 		/*loopback;*/
 		#phy-cells	= <0>;
 		reg		= <1>;
-- 
1.9.1

